neo/Settings.py

```diff
@@ -309,5 +320,5 @@ def check_privatenet(self):
 
 # Check if currently installed dependencies match the requirements
 # Can be bypassed with `SKIP_DEPS_CHECK=1 python prompt.py`
-if not os.getenv("SKIP_DEPS_CHECK"):
+if not os.getenv("SKIP_DEPS_CHECK") and not IS_PACKAGE_INSTALL:
```

Comment:
Why would you skip the dependency check when installed as package?

Reply:
Requirements.txt isn't included in the package, so we can't read it.  Also, If we're running from a package wouldn't it be safe to assume that the bundled code would have the correct dependencies?

Reply:
yeah good point, that actually should work just fine. but then again, wouldn't the check work anyway? ^^ but yeah, this is mostly intended for people pulling from the dev branch, so disabling it for using as package is fine for me. there might even be weird conflicts where the user updates a specific package and neo-python would scream because it's not the exact version it expects, so perhaps even better like this. 👍